packages/mui-base/src/Switch/Switch.tsx

```diff
@@ -126,7 +135,7 @@ const Switch = React.forwardRef(function Switch<RootComponentType extends React.
   });
 
   return (
-    <Root {...rootProps}>
+    <Root {...rootProps} sx={{ outline: 'none' }}>
```

Comment:
Base UI components do not support the `sx` prop. What was your intention here?

Reply:
oh sorry, forgot to delete it.